developer_manual/app/info.rst

```diff
@@ -127,19 +127,31 @@ ownCloud allows to specify four kind of ``types``. Currently supported ``types``
 
 documentation
 -------------
-Link to 'admin', 'user', 'developer' documentation
+**Required**: Link to 'admin', 'user' and 'developer' documentation
```

Comment:
This PR depends on https://github.com/owncloud/documentation/pull/2471#issuecomment-224827633 which has open comments